com.microsoft.mrtk.input/Controllers/ArticulatedHandController.cs

```diff
@@ -116,5 +141,36 @@ protected override void UpdateInput(XRControllerState controllerState)
                 }
             }
         }
+
+        /// <inheritdoc />
+        protected override void UpdateTrackingInput(XRControllerState controllerState)
+        {
+            base.UpdateTrackingInput(controllerState);
+
+            // Workaround for missing aim/pointing-pose on devices without interaction profiles
+            // for hands, such as Varjo and Quest. Should be removed once we have universal
+            // hand interaction profile(s) across vendors.
+            if (handsAggregator != null && (positionAction.action?.controls.Count ?? 0) == 0)
+            {
+                if (!handsAggregator.TryGetJoint(TrackedHandJoint.IndexProximal, HandNode, out HandJointPose knuckle))
+                {
+                    return;
+                }
+                if (!handsAggregator.TryGetJoint(TrackedHandJoint.Palm, HandNode, out HandJointPose palm))
+                {
+                    return;
+                }
+
+                // Tick the hand ray generator function. Uses index knuckle for position.
+                handRay.Update(PlayspaceUtilities.ReferenceTransform.TransformPoint(knuckle.Position),
+                               PlayspaceUtilities.ReferenceTransform.TransformVector(-palm.Up),
+                               CameraCache.Main.transform,
+                               HandNode.ToHandedness());
+                
+                Ray ray = handRay.Ray;
+                controllerState.position = ray.origin;
+                controllerState.rotation = Quaternion.LookRotation(ray.direction, PlayspaceUtilities.ReferenceTransform.TransformVector(palm.Up));
```

Comment:
Interesting...everywhere else I could find just uses `Quaternion.LookRotation(ray.direction);`
Does this account for a case the others are missing?